src/Storages/StorageFile.cpp

```diff
@@ -793,6 +791,12 @@ StorageFile::StorageFile(const std::string & table_path_, const std::string & us
     setStorageMetadata(args);
 }
 
+StorageFile::StorageFile(const std::string & table_path_, const std::string & user_files_path, bool distributed_processing_, CommonArguments args)
+    : StorageFile(table_path_, user_files_path, args)
+{
+    distributed_processing = distributed_processing_;
+}
```

Comment:
```suggestion
StorageFile::StorageFile(const std::string & table_path_, const std::string & user_files_path, bool distributed_processing_, CommonArguments args)
    : StorageFile(table_path_, user_files_path, args), distributed_processing(distributed_processing_)
{
}
```

Reply:
`An initializer for a delegating constructor must appear alone`